src/elements/content-sidebar/SidebarContent.scss

```diff
@@ -35,12 +40,20 @@
 
     .bcs-scroll-content {
         // Styles can be overriden by each sidebar
-        width: 323px; // 17px less than the total width for scroll bar
+        width: $sidebarScrollContentWidth; // 17px less than the total width for scroll bar
+    }
+}
+
+.bcs-wider .bcs-content {
+    width: $sidebarIncreasedWidth;
+
+    .bcs-scroll-content {
+        width: $sidebarScrollContentIncreasedWidth;
     }
 }
 
 @include breakpoint($medium-screen) {
-    .bcs-content {
+    .bcs-wider .bcs-content {
```

Comment:
I think we should have two options here: 
1. .bcs-content  
2. .bcs-wider .bcs-content
cause now it seems we removed the first one and there are no styles for regular-width content in medium viewport